drivers/spi/spi_ll_stm32.c

```diff
@@ -341,17 +342,20 @@ static int spi_stm32_get_err(SPI_TypeDef *spi)
 /* Shift a SPI frame as master. */
 static void spi_stm32_shift_m(SPI_TypeDef *spi, struct spi_stm32_data *data)
 {
+	unsigned int frame_bytes = bits2bytes(
```

Comment:
could it make sense to just call it dfs (aka Data Frame Size) like it is done in many other drivers? (unless ST do not use that term in their specs, I don't know)